src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -108,7 +105,7 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
                     final Entity entityCitizen = world.getEntityByID(citizenDataView.getEntityId());
                     if (entityCitizen instanceof EntityCitizen && entityCitizen.getPosition().distanceSq(player.getPosition()) <= 2)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2589)